playwright/mockData/orgs/KPD/people/views/AllMembers/columns.ts

```diff
@@ -0,0 +1,18 @@
+import { ZetkinViewColumn } from '../../../../../../../src/types/zetkin';
```

Comment:
Very small thing, but if we have many view types, we may want to move these to their own file, like how we did `types/smartSearch.ts`